Assignment_1/palindrome_permutation.js

```diff
@@ -0,0 +1,33 @@
+const palindromePermutation = (str) => {
+
```

Comment:
The convention in JS is to not have this extra line of whitespace after an opening bracket.